src/electron/process_manager.ts

```diff
@@ -166,19 +183,16 @@ export class TunnelManager {
     }
 
     // ss-local must be up in order to test UDP support and validate credentials.
-    this.ssLocal.start(this.config);
-    await isSsLocalReachable();
+    await this.shadowsocks.start();
 
     // Don't validate credentials on boot: if the key was revoked, we want the system to stay
     // "connected" so that traffic doesn't leak.
+    // TODO(alalama): pass UDP support on auto-connect, currently we fallback to TCP by default.
```

Comment:
What does this mean? Do we lose UDP support on auto connect?

That could explain https://github.com/Jigsaw-Code/outline-client/issues/878

Reply:
Yes, however this may be correct because we don't know whether the (new) network supports UDP. 
I don't think it's entirely related to that issue, because the user re-connected, which would trigger the UDP check.